common-arcana.lic

```diff
@@ -72,35 +72,42 @@ module DRCA
     end
   end
 
-  def start_barb_abilities(skills, settings)
-    skills
+  def start_barb_abilities(abilities, settings)
+    # Note, you must know Power meditation or Powermonger mastery
+    # for your active abilities to be detected by DRSpells.
+    abilities
       .reject { |name| DRSpells.active_spells[name] }
-      .each do |name|
-        activate_barb_buff?(name)
-        waitrt?
-        pause settings.meditation_pause_timer.to_i if get_data('spells').barb_abilities[name]['type'].eql?('meditation')
-        DRC.fix_standing
-      end
+      .each { |name| activate_barb_buff?(name, settings) }
+    DRC.fix_standing
   end
 
-  def activate_barb_buff?(name)
+  def activate_barb_buff?(name, settings = nil)
```

Comment:
Add new, optional argument for backwards compatibility with combat-trainer. Once this PR is merged and then after #4762 then we can decide if this second argument should remain optional or not. As of this PR, the settings is only used if activating a meditation so the script can know how long to pause.